docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -393,12 +421,19 @@ Start by adding the following dependencies to your test project:
 testImplementation("io.rest-assured:rest-assured")
 testImplementation("io.quarkus:quarkus-junit5")
 ----
+====
 
-[[integration-testing-wiremock]]
-==== Wiremock
+//@sberyozkin - It might be good to add a stament here about the different mechanisms or methods to integration testing. For example, WireMock... etc
 
-Add the following dependencies to your test project:
 
+[[wiremock-integration-testing]]
```

Comment:
I'm not sure the label name change is necessary, there might already be be external links pointing to it (from the code flow docs for example, from somewhere else), other labels follow a different pattern, it all can be realigned but it is not worth it IMHO :-)

Reply:
@sberyozkin - Good point. Fixed. I think I was trying to fix a broken anchor, and somehow I decided to recreate it this way :thinking:  I have reverted it back and kept the anchor fix in but with the original ID. Thanks!